{"name": "test_alert3_failure", "status": "passed", "steps": [{"name": "Нажатие кнопки для вызова Alert 3", "status": "passed", "start": 1760096660093, "stop": 1760096660209}, {"name": "Отменить Cancel у Alert 3", "status": "passed", "start": 1760096660211, "stop": 1760096660217}, {"name": "Подтверждаем, что после нажатия Отмены у нас выпало нужное предложение", "status": "passed", "steps": [{"name": "Получить текст после нажатия Alert 3", "status": "passed", "start": 1760096660260, "stop": 1760096660283}], "start": 1760096660217, "stop": 1760096660283}], "start": 1760096599317, "stop": 1760096660283, "uuid": "193fe4fc-603b-4e15-9e56-6a00c09f2a1c", "historyId": "168378ac3bb54ecc85339052d470cf0c", "testCaseId": "168378ac3bb54ecc85339052d470cf0c", "fullName": "test_alerts.TestAlertsPage#test_alert3_failure", "labels": [{"name": "feature", "value": "Alerts"}, {"name": "story", "value": "Тест второго Alert (Негативный - нажатие Отмены)"}, {"name": "suite", "value": "test_alerts"}, {"name": "subSuite", "value": "TestAlertsPage"}, {"name": "host", "value": "DESKTOP-T28TT8C"}, {"name": "thread", "value": "11056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alerts"}], "titlePath": ["test_alerts.py", "TestAlertsPage"]}